src/test/resources/com/owlcyberdefense/cobol1/tests.tdml

```diff
@@ -0,0 +1,85 @@
+    <document>
+      <documentPart type="text" encoding="ebcdic-cp-us">1234567890123456789</documentPart>
+      <documentPart type="byte">0020161C</documentPart>
+      <documentPart type="byte">0020161C</documentPart>
+      <documentPart type="byte">0020161C</documentPart>
+      <documentPart type="byte">0020161C</documentPart>
+      <documentPart type="byte">99999C</documentPart>
+      <documentPart type="byte">270f</documentPart>
+      <documentPart type="byte">000f423f</documentPart>
+      <documentPart type="byte">00000002540be3ff</documentPart>
+      <documentPart type="text" encoding="ebcdic-cp-us">PROVIDER12345</documentPart>
+      <documentPart type="byte">0999999C</documentPart>
+      <documentPart type="byte">0999999C</documentPart>
+      <!-- 
+      The decimal point character in the string below should NOT be in the
+      data. It is there for now because Daffodil does not yet implement the "V"
+      character for dfdl:textNumberPattern indicating the location of an 
+      implied decimal point. 
+       -->
+      <documentPart type="text" encoding="ebcdic-cp-us">99999999y</documentPart>
+      
+      <documentPart type="text" encoding="ebcdic-cp-us">AA</documentPart>
+      <documentPart type="byte">0001869f</documentPart>
+      <documentPart type="byte">270f</documentPart>
+      <documentPart type="byte">03e7</documentPart>
+      <documentPart type="byte">F9</documentPart>
+      <documentPart type="byte">F8</documentPart>
+      <documentPart type="byte">F7</documentPart>
+      <documentPart type="text" encoding="ebcdic-cp-us">X</documentPart>
+      <documentPart type="text" encoding="ebcdic-cp-us">A12345678901234567890AB</documentPart>
+    </document>
+    <infoset>
+      <dfdlInfoset>
+          <ex:MY-RECORD xmlns="">
+           <CLAIM-NUMBER>1234567890123456789</CLAIM-NUMBER>
+        <ADMISSION-DATE>1961-02-01</ADMISSION-DATE>
+        <FROM-DATE>1961-02-01</FROM-DATE>
+        <THRU-DATE>1961-02-01</THRU-DATE>
+        <DISCHARGE-DATE>1961-02-01</DISCHARGE-DATE>
+        <FULL-DAYS>99999</FULL-DAYS>
+        <COINSURANCE-DAYS>9999</COINSURANCE-DAYS>
+        <LIFETIME-RES-DAYS>999999</LIFETIME-RES-DAYS>
+        <INTERMEDIARY-NUM>9999999999</INTERMEDIARY-NUM>
+        <PROVIDER>PROVIDER12345</PROVIDER>
+        <INPATIENT-DED>9999.99</INPATIENT-DED>
+        <BLOOD-DED>9999.99</BLOOD-DED>
+        <TOTAL-CHARGES>-9999999.99</TOTAL-CHARGES>
+        <PATIENT-STATUS>AA</PATIENT-STATUS>
+        <BLOOD-PINTS-FURNISHED>99999</BLOOD-PINTS-FURNISHED>
+        <BLOOD-PINTS-REPLACED>9999</BLOOD-PINTS-REPLACED>
+        <SEQUENCE-COUNTER>999</SEQUENCE-COUNTER>
+        <BILL-SOURCE>9</BILL-SOURCE>
+        <BENEFITS-EXHAUST-IND>8</BENEFITS-EXHAUST-IND>
+        <BENEFITS-PAY-IND>7</BENEFITS-PAY-IND>
+        <AUTO-ADJUSTMENT-IND>X</AUTO-ADJUSTMENT-IND>
+        <INTERMEDIARY-CTRL-NUM>A12345678901234567890AB</INTERMEDIARY-CTRL-NUM>
+          </ex:MY-RECORD>
+      </dfdlInfoset>
```

Comment:
Consider re-indenting this file with an IDE (the indentation doesn't look consistent).